portstyle.css

```diff
@@ -0,0 +1,229 @@
+	z-index: 10;
+	border-bottom: 2px solid #c67e59;
+}
+
+.navigationBar a {
+	margin: 20px auto;
+	font-size: 1.5rem;
+	background: none;
+	border: none;
+	text-decoration: none;
+}
+
+
+.mainImage {
+	display: flex;
+	justify-content: center;
+	width: 100%;
+}
+
+
+.backgroundImage {
+	background-image: url("https://images.unsplash.com/photo-1592750088063-96b411cd5895?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80");
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	height: 100vh;
+	width: 100%;
+	z-index: -20;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.overlayText {
+	position: fixed;
+	border: 1px solid white;
+	padding: 0px 1%;
+	z-index: -10;
+}
+
+.overlayText h1 {
+	display: flex;
+	justify-content: center;
+	font-size: 1.5rem;
+}
+
+
+.aboutMe {
+	display: grid;
+	grid-template-columns: 50% 50%;
+	padding: 200px 20% 280px 20%;
+	column-gap: 20px;
+	background: #78a6b8;
+}
+
+
+.aboutMeBox {
+	display: flex;
+	padding-left: 20px;
+	flex-direction: column;
+}
+
+.aboutMeTitle {
+	display: flex;
+	align-self: center;
+}
+
+.pictureOfOwner {
+	width: 100%;
+	height: 100%;
+	align-self: center;
+}
+
+
+.textAboutMe {
+	display: flex;
+	align-items: center;
+}
+
+.portfolioTitle {
+	display: flex;
+	justify-content: center;
+	padding-top: 50px;
+	background: #658fa7;
+}
+
+
+.threeBoxLayout {
+	display: grid;
+	grid-template-columns: 33.3% 33.3% 33.3%;
+	padding: 10px 20% 100px 20%;
+	background: #658fa7;
+}
+
+
+.threeBoxLayout .portfolioBox {
+	display: flex;
+	padding: 20px 20px;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.background {
+	display: grid;
+	grid-template-rows: 1fr 1fr;
+	background: #78a6b8;
+	align-items: flex-start;
+	border: 1px solid #c67e59;
+	height: 400px;
+}
+
+.background img {
+	object-fit: cover;
+	height: 200px;
+	width: 100%;
+}
+
+
+.portfolioText {
+	display: flex;
+	justify-content: center;
+	text-decoration: none;
+	color: #FFFFFF;
+	
+}
+
+.portfolioText a {
+	text-decoration: none;
+}
+
+
+footer {
+	display: grid;
+	grid-template-rows: 20% 80%;
+	width: 100%;
+	background: #22303a;
+	border-top: 2px solid #c67e59;
+}
+	
+.socialsHeader {
+	justify-self: center;
+	align-self: center;
+	margin-top: 20px;
+}
+
+.socialLinksActual {
+	display: flex;
+	justify-content: center;
+	display: grid;
+	grid-template-columns: 10% 10% 10%;
+	column-gap: 10px;
+	margin: 20px 10% 10px 10%;
+	
+}
+
+.socialLinksActual a {
+	display: grid;
+}
+
+.mediaLogo {
+	width: 100%;
+	height: 70%;
+	place-self: center;
+}
+
+@media screen and (max-width: 1100px) {
+	
+	.threeBoxLayout {
+		display: flex;
+		width: 100%;
+		grid-template-columns: 100%;
+		flex-direction: column;
+		
+	}
+}
+	
+@media screen and (max-width: 1100px) {
+	.aboutMe {
+			display: grid;
+			grid-template-columns: 1fr;
+		}
+}
+	
+
+@media screen and (max-width: 400px) {
+	.navigationBar a {
+		padding: 3px;
+		margin: 1%;
+		
+	}
+	.overlayText {
+		top: 41%;
```

Comment:
why? what is the purpose of this?

Reply:
It was originally to keep the .overlayText in place for when it was shrunk down to mobile. I have just tried removing it and it works anyway so I ahve removed.